lib/screens/departures/departure.ex

```diff
@@ -170,7 +170,7 @@ defmodule Screens.Departures.Departure do
     filtered_predictions = filter_predictions_or_schedules(all_predictions)
     filtered_schedules = filter_predictions_or_schedules(all_schedules)
 
-    filtered_predictions = copy_stop_headsigns(filtered_predictions, filtered_schedules)
+    filtered_predictions = copy_stop_headsigns(filtered_predictions, all_schedules)
```

Comment:
The departures changes should have been a new branch, since it wasn't a bug introduced in your work. (Includes the v1/v2 updates and test edits)